x-pack/plugins/uptime/server/rest_api/certs/certs.ts

```diff
@@ -9,10 +9,10 @@ import { API_URLS } from '../../../common/constants';
 import { UMServerLibs } from '../../lib/lib';
 import { UMRestApiRouteFactory } from '../types';
 
-const DEFAULT_INDEX = 0;
+export const DEFAULT_FROM = 'now-15m';
+export const DEFAULT_TO = 'now';
```

Comment:
this is also going to effect cert displayed.

Reply:
++ this is intentional. The alert and the display should stay aligned. If you get an alert, and see different data on the page, that will be confusing.